luxer_sale_subscription/models/sale_order.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    site_address = fields.Many2one(comodel_name='res.partner')
+
+    def _prepare_invoice(self):
+        invoice_vals = super(SaleOrder, self)._prepare_invoice()
+        invoice_vals['site_address'] = self.site_address.id
```

Comment:
might break if there is no site_address set
try to do something like this: https://github.com/odoo/enterprise/blob/16.0/sale_subscription/models/sale_order.py#L495